student-hub-app/src/Resume/restumeDoc.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react'
+import './restumeDoc.css';
+
+class restumeDoc extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      school: '',
+      major:'',
+      experience: '',
+      skills: []
+    }
+  }
+
+  handleInputChange = (e) => {
+    this.setState({ [e.target.name]: e.target.value })
+  }
+
+  addSkill = () => {
+    this.setState({ 
+      skills: [...this.state.skills, this.state.currentSkill], 
+      currentSkill: '' 
+    })
+  }
+
+  removeSkill = (indexToRemove) => {
+    this.setState({ 
+      skills: this.state.skills.filter((_, index) => index !== indexToRemove) 
+    })
+  }
+
+  render() {
+    const { experience, school, major, skills, currentSkill } = this.state
+    return (
+      <div>
+        <label>Experience:</label>
+        <input 
+          name='experience' 
+          value={experience} 
+          onChange={this.handleInputChange} 
+          placeholder='Enter your work experience' 
+          />
+        <label>School:</label>
+        <input 
+          name='school' 
+          value={school} 
+          onChange={this.handleInputChange} 
+          placeholder='Enter your education details' 
+          />
+        <label>Major:</label>
+        <input 
+          name='major' 
+          value={major} 
+          onChange={this.handleInputChange} 
+          placeholder='Enter your major' 
+          />
+        <label>Skills:</label>
+        { skills.map((skill, index) => (
+          <div key={index}>
+            {skill}
+            <button onClick={() => this.removeSkill(index)}>Remove</button>
+          </div>
+        ))}
+        <input 
+          value={currentSkill} 
+          onChange={e => this.setState({ currentSkill: e.target.value })} 
+        />
+        <button onClick={this.addSkill} >Add Skill</button>
+      </div>
+    )
+  }
```

Comment:
This questionnaire ordering makes very little sense. In theory, it should first be General info, then skills, then job experience (with the ability to add multiple jobs) info, then projects (same deal) then extracurricular, and for each of these you will need to talk with Owen or reference the DB to see what fields are mandatory and what values each field needs